pages/privacy-basics.jsx

```diff
@@ -0,0 +1,194 @@
+    return (
+      <section className="row privacy-basics">
+        <div className="col-sm-12">
+          <h2>Learning Objectives</h2>
+          <p>
+            Learners will understand Web mechanics, security, and privacy as they analyze and reflect on common surveillance practices, as well as their own privacy habits.
+          </p>
+        </div>
+      </section>
+    );
+  }
+});
+
+
+var PrivacyBasics = React.createClass({
+  statics: {
+    pageTitle: 'Privacy Basics: Passwords, Tracking, and Data Retention',
+    pageClassName: 'privacy-basics'
+  },
+  sections: [
+    {
+      title: "Passwords",
+      activities: [
+        {
+          title: "Passwords Agenda",
+          image1x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          image2x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          subtitle: "Understanding Security",
+          description: "Learners will evaluate strong and weak passwords and develop their own strategies for creating secure passwords, learning about security and privacy.",
+          link: "https://chadsansing.github.io/curriculum-testing/expanded-privacy-curriculum/passwords.html"
+        },
+        {
+          title: "Testing Password Strength",
+          image1x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          image2x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          subtitle: "Understanding Security",
+          description: "Learners will look at leaked passwords to identify weak password habits to avoid and then they'll test possible passwords using an online \"password meter\", learning about security and privacy.",
+          link: "https://chadsansing.github.io/curriculum-testing/expanded-privacy-curriculum/testing-password-strength.html"
+        },
+        {
+          title: "Creating Strong Passwords",
+          image1x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          image2x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          subtitle: "Understanding Security",
+          description: "Learners will use several different ways to create strong passwords, learning about security and privacy.",
+          link: "https://chadsansing.github.io/curriculum-testing/expanded-privacy-curriculum/creating-strong-passwords.html"
+        },
+        {
+          title: "Avoiding Password Reuse",
+          image1x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          image2x: " https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png",
+          subtitle: "Understanding Security",
+          description: "Learners will see why re-using passwords is a bad idea and pick up strategies for avoiding that habit, learning about security.",
+          link: "https://d157rqmxrxj6ey.cloudfront.net/mozillalearning/11704/"
```

Comment:
should point to http://chadsansing.github.io/curriculum-testing/expanded-privacy-curriculum/avoiding-password-reuse.html